app/src/main/java/com/thinqtv/thinqtv_android/GetInvolved.java

```diff
@@ -7,22 +7,92 @@
 import android.content.Intent;
 import android.os.Bundle;
 import android.view.View;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Spinner;
 
-public class GetInvolved extends AppCompatActivity {
+public class GetInvolved extends AppCompatActivity implements AdapterView.OnItemSelectedListener {
+    private WebView view;
```

Comment:
Can you rename this variable to be more specific? Also, is there any reason for it to be an instance variable rather than a local variable?

Reply:
Actually... maybe it doesn't need to be renamed if it's the only view being used in this class. I'll leave it up to you.